app/knight_function.py

```diff
@@ -0,0 +1,64 @@
+from typing import Dict, Union
+
+
+def prepare_knight(
+        knight: Dict[
+            str, Union[
+                str, int, list, Dict[
+                    str, Union[
+                        str, int, list, Dict[str, int]
+                    ]
+                ]
+            ]
+        ]
+) -> Dict[str, Union[str, int]]:
```

Comment:
type hint is difficult to understand, simplify it
```suggestion
def prepare_knight(knight: Dict) -> Dict[str, Union[str, int]]:
```